net/proxy_resolution/pac_file_fetcher_impl_unittest.cc

```diff
@@ -171,7 +171,7 @@ TEST_F(PacFileFetcherImplTest, HttpMimeType) {
                                     TRAFFIC_ANNOTATION_FOR_TESTS);
     EXPECT_THAT(result, IsError(ERR_IO_PENDING));
     EXPECT_THAT(callback.WaitForResult(), IsOk());
-    EXPECT_EQ(u"-pac.txt-\n", text);
+    EXPECT_EQ(text.compare(0, 9, u"-pac.txt-"), 0);
```

Comment:
Why are the tests changing here ?

Reply:
The test fails because the received string is "-pac.txt-\r\n", a typical Windows newline.